src/proto/grpc/extension/extension.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package grpc.extension;
+
+message ClientConfig {
+  repeated ChannelConfig client_channel = 1;
+}
+
+message ChannelConfig {
```

Comment:
ChannelManagementConfig

Reply:
Need to keep this as ChannelConfig. As retry, timeout, regional will be added here.